linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/tests/config_and_current_same_time.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# platform = Red Hat Enterprise Linux 8
+
+# This is false negative scenario
```

Comment:
@matusmarhefka @mildas Do you see value in this test scenario?
Strictly speaking, this should be a `fail` scenario. But AFAIK, there is no way to detect this, as smallest time unit in OVAL is one second.

Reply:
I see, if we want to fake the files to have the same modification timestamps I would do the following trick:
```
update-crypto-policies --set "DEFAULT"
CDATE=$(date +%Y%m%d%H%M)
touch -a -m -t "$CDATE" /etc/crypto-policies/config /etc/crypto-policies/state/current
```

Reply:
This is strictly to just test the situation when file modification time is the same, but I understand that this is OVAL limitation and we cannot really catch situations when modification time difference is less than 1s.

Reply:
Thanks for the tip for the timestamp, I'll add this to the test scnerio.
But my concern was more about whether we should keep a test like this, to illustrate the situation we cannot cover well.

Reply:
I don't know what you mean with `false negative scenario` in this case (shouldn't it be `false positive`? but probably I'm just confused). However, I like this test scenario and I see value in it because we can test that this specific case is passing and not causing failures from <1sec roundings.

The next comment about precision of seconds is very important there and probably I would add to it something like `WARNING:` or `IMPORTANT:` just to emphasize its importance.

Reply:
@mildas Hmm, I think it is a false negative, as it reports a no-issue situation where there is one.

> However, I like this test scenario and I see value in it because we can test that this specific case is passing and not causing failures from <1sec roundings.

I think the rounding issue is solved in 3df3833f8191f8c261d02f9dd958a23f2619bfbb, it was caused by comparing the timestamp of the file to the current time.

In this test scenario, the <1sec difference is an actual issue.